gulp/.eslintrc

```diff
@@ -1,5 +1,6 @@
 {
   "rules": {
-   "no-restricted-syntax": 0
+   "no-restricted-syntax": 0,
+   "no-console": 0
```

Comment:
Was this included intentionally?

Reply:
This is the eslintrc for just the gulp files, I think it's fine.